config/locales/en.yml

```diff
@@ -177,11 +177,13 @@ en:
           Scholarships of £27,000 are available for trainee French, German, and Spanish language teachers.
 
           Bursaries of £25,000 are available for all trainee language teachers, including ancient languages.
+
+          Non-UK citizens without settled status in the UK are still eligible for this support.
```

Comment:
Are non-UK citizens eligible for other funding for languages? If not, do we need to make this clear (or will it be clear on the destinations pages so that is sufficient?)
![image](https://github.com/DFE-Digital/get-into-teaching-app/assets/87642394/cb9ac750-441d-4ce5-8cd9-45e0e5ad6ff0)

Reply:
Same question re loans and other funding for Physics!